packages/sdk/client-services/src/packlets/services/service-host.ts

```diff
@@ -117,11 +131,38 @@ export class ClientServicesHost {
     return this._serviceRegistry.services;
   }
 
+  /**
+   * Initialize the service host with the config.
+   * Config can also be provided in the constructor.
+   * Can only be called once.
+   */
+  initialize({ config, networkManager }: InitializeOptions) {
```

Comment:
why can this only be called once? seems like it would be simpler if it could just be a part of `open`?

Reply:
Passing in config, and networkManager over and over again when `open` and `close` are called multiple times feels kinda wrong. Im thinking `initialize` is to setup/configure the host and then `open` and `close` are the lifecycle.